test/ussd_registration.test.js

```diff
@@ -587,6 +598,17 @@ describe("Mama Nigeria App", function() {
                         state: 'state_end_sms',
                         reply: "Thank you. The person will now start receiving messages three times a week."
                     })
+                    .check(function(api) {
+                        var expected_used = [1,6,36,37,38,46,54,59,60,61];
+                        var fixts = api.http.fixtures.fixtures;
+                        var i = 0;
+                        var fixts_used = [];
+                        fixts.forEach(function(f) {
```

Comment:
`forEach` already has an index param that negates the need for `i` etc. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach